addons/event_sale/models/event.py

```diff
@@ -50,6 +52,9 @@ class Event(models.Model):
         'res.currency', string='Currency',
         default=lambda self: self.env.company.currency_id.id, readonly=True)
 
+    start_sale_date = fields.Date('Start sale date', compute='_compute_start_sale_date')
+    event_registrations_open = fields.Boolean('Registration open', compute='_compute_event_registrations_open')
```

Comment:
Why redefining a field defined in event ? You can just override _compute_event_registrations_open in event_sale without redefining the field.